config/tests/samples/create/harness.go

```diff
@@ -649,7 +649,8 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 	if os.Getenv("E2E_GCP_TARGET") == "vcr" {
 		// TODO(yuhou): use a cleaner way(resource kind) to manage the allow list for vcr
 		switch name {
-		case "fullalloydbcluster":
+		// update test data requires regeneration of the vcr log, skip the test for now.
+		// case "fullalloydbcluster":
```

Comment:
@gemmahou do you want to ensure we always have this test case enabled here?

-> If so, could you (1) help @vmiglani figure out how to do it, (2) add proper guide in https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md and other relevant docs, and (3) add proper unit tests to prevent contributors from commenting out the test cases?

-> If not, can we also have proper docs about how we handle it? Maybe start with always adding a tracking issue for the commented out test cases?

Also a side note, please do not use your internal username in the TODO for OSS project. Could you clean them up when you get a chance?

Reply:
For new resources, it's not required to add vcr logs. I plan to have a more automation way to update vcr logs periodically. 

In this PR, external contributer had to modify the test data that we enabled vcr test and captured vcr logs using the old test data. Ideally we suggest external contributer to update the vcr log as well. I chatted with @vmiglani offline about this. We have a guide on how to capture VCR logs, see this [readme](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/README.VCRTesting.md).

But Vandana encountered local building issue and she was not able to run the unified test(that we used to capture vcr log) and samples test locally. To unblock the PR, I suggest her to remove this test from the list for now, and once this PR is merged, I will update the log and re-enable the test.